lib/json1.ts

```diff
@@ -688,7 +689,11 @@ function transformPosition(path: Path, op: JSONOp): Path | null {
       if (et) {
         const e = getEdit(c!)
         log('Embedded edit', e, et)
-        if (et.transformPosition) path[i] = et.transformPosition(path[i], e)
+        if (et.transformPosition) {
+            path[i] = et.transformPosition(path[i], e)
+        } else if (et.transformCursor) {
```

Comment:
Where would `transformCursor` come from? Is this for rich text?

Reply:
Exactly, not ideal but they renamed it there: https://github.com/ottypes/rich-text/blob/master/lib/type.js#L39-L41

Reply:
Ah I see.

Would it work to patch rich text externally with something like `richText.transformPosition = richText.transformCursor`, and leave this code here unchanged?

Reply:
Actually, I got it backward. As per [the main OTType documentation](https://github.com/ottypes/docs/blob/master/README.md#optional-properties) it's `transformCursor` that the type should implement. And it's `text-unicode` (the default subtype in JSON1), that is missing it but implements [`transformPosition`](https://github.com/ottypes/text-unicode/blob/master/lib/type.ts#L446-L474).